docs/features/dx_synth.md

```diff
@@ -93,6 +93,24 @@ However when feedback is activated on algorithm 4 and 6 (those will multi-operat
 "auto" will instead use the "vintage" engine as the modern one doesn't implement these algorithms properly.
 "vintage" engine is meant to more accurately model a DX7 synth, but has lower polyphony.
 
+## MIDI integration
+
+The synth supports sending and receiving param edits as sysex messages, in the standard format supported by DX7 and many
+software implementations such as dexed. It is possible to use the Dexed UI for editing patch parameters.
+Here is an example for how setting up [Dexed](https://asb2m10.github.io/dexed/)
+
+- Connect the Deluge to the computer with USB
+- Start dexed in "Standalone" mode.
+- Click the "options" button in the top left.
+    - Make sure "Deluge MIDI 1" is selected as an "Active MIDI input". Also set "MIDI output" to "Deluge MIDI 1"
```

Comment:
these instructions should probably go to midi port 3 - we don't implement midi sysex stream interruption so I think it could cause bugs if you're playing notes and editing at the same time